core/src/main/scala/com/mesosphere/usi/core/matching/OfferMatcher.scala

```diff
@@ -38,14 +40,27 @@ class OfferMatcher(masterDomainInfo: Mesos.DomainInfo) {
       result: Map[RunningPodSpec, List[OfferMatcher.ResourceMatch]],
       pendingLaunchPodSpecs: List[RunningPodSpec]): Map[RunningPodSpec, List[OfferMatcher.ResourceMatch]] = {
 
+    def matchesAgentFilter(agentFilters: Iterable[AgentFilter]): Boolean = {
```

Comment:
```suggestion
    def matchesAgentFilters(agentFilters: Iterable[AgentFilter]): Boolean = {
```

Just me nit pickting.